libraries/platforms/bigblade-vcs/execution.mk

```diff
@@ -47,6 +47,9 @@ debug.log: $(BSG_MACHINE_PATH)/$(BSG_PLATFORM)/debug/simv
 exec.log: $(BSG_MACHINE_PATH)/$(BSG_PLATFORM)/exec/simv
 profile.log: $(BSG_MACHINE_PATH)/$(BSG_PLATFORM)/profile/simv
 
+stats: profile.log
+	PYTHONPATH=/home/drichmond/Research/repositories/git/bsg_bladerunner/bsg_manycore/software/py/ python3 -m vanilla_parser --only stats_parser --stats vanilla_stats.csv  --vcache-stats vcache_stats.csv  --tile-group --tile --cache-line-words $(BSG_MACHINE_VCACHE_LINE_WORDS)
```

Comment:
Path?

Reply:
Yeah, my bad. Fixed (with an override for the other files, too)